packages/proxyIdentity/contracts/OfferableIdentity.sol

```diff
@@ -1,19 +1,33 @@
-pragma solidity 0.8.0;
+pragma solidity 0.4.26;
 
+import "./IERC165.sol";
 import "./IdentityManager.sol";
 
 
-contract OfferableIdentity {
+contract OfferableIdentity is IERC165 {
```

Comment:
Why using our own `IERCI65`  interface instead of the OpenZeppelin `IERC165` interface or (perhaps even better) inheriting from the `ERC165` contract? https://docs.openzeppelin.com/contracts/4.x/utilities#api:introspection.adoc#ERC165